Extensions/NotificationService/NotificationService.swift

```diff
@@ -174,18 +174,18 @@ class SyncDataDisplay {
     }
 
     func displayNewSentTabNotification(tab: [String: String]) {
-        if let urlString = tab["url"],
+        if let urlString = tab[NotificationSentTabs.Payload.urlKey],
             let url = URL(string: urlString, invalidCharacters: false),
             url.isWebPage(),
-            let title = tab["title"] {
+            let title = tab[NotificationSentTabs.Payload.titleKey] {
             let tab = [
-                "title": title,
-                "url": url.absoluteString,
-                "displayURL": url.absoluteDisplayExternalString,
-                "deviceName": nil
+                NotificationSentTabs.Payload.titleKey: title,
+                NotificationSentTabs.Payload.urlKey: url.absoluteString,
+                NotificationSentTabs.Payload.displayURLKey: url.absoluteDisplayExternalString,
+                NotificationSentTabs.Payload.deviceNameKey: nil
             ] as NSDictionary
 
-            notificationContent.userInfo["sentTabs"] = [tab] as NSArray
+            notificationContent.userInfo[NotificationSentTabs.sentTabsKey] = [tab] as NSArray
```

Comment:
We could change the one in SceneDelegate as well?

Reply:
That one is updated also in this PR ([link](https://github.com/mozilla-mobile/firefox-ios/pull/17533/files#diff-67a8aa8c79483767108e53c9f01c21b9f7aedbbc18d8bd8448c556bdbba633d3)) but LMK if you are referring to something else, happy to change further